Detectors/TOF/calibration/src/TOFChannelCalibrator.cxx

```diff
@@ -609,7 +609,7 @@ void TOFChannelCalibrator<T>::finalizeSlotWithCosmics(Slot& slot)
 
   auto clName = o2::utils::MemFileHelper::getClassName(ts);
   auto flName = o2::ccdb::CcdbApi::generateFileName(clName);
-  mInfoVector.emplace_back("TOF/Calib/ChannelCalib", clName, flName, md, slot.getTFStart(), o2::ccdb::CcdbObjectInfo::INFINITE_TIMESTAMP);
+  mInfoVector.emplace_back("TOF/Calib/ChannelCalib", clName, flName, md, slot.getStartTimeMS(), o2::ccdb::CcdbObjectInfo::INFINITE_TIMESTAMP);
```

Comment:
does this really need INFINITE_TIMESTAMP ?

Reply:
Yes, I know. The problem are long TS, also over Xmas. We don't want to restart from scratch. So either we set here 1 month (it should be enough for TSs, right? unless something breaks dramatically), and then do a manual upload when data taking restarts (but we need to remember it!), or better this limit. Even if it will not be convenient...
Maybe we could use 1 year?

Reply:
If this is frequently updated object, then even 1 year will create lot of overlaps (and if it indeed changes so often, is 1 year old object any better than the nominal default one?). I would consider 1-2 months margin at most, and if needed, would extend the validity of last (non-default) entry, perhaps we should have a generic tool for that, which check a list of objects for which such operation makes sense.

Reply:
Now 2 months are used.